pkg/plugin/plugin_manager.go

```diff
@@ -51,7 +51,15 @@ func LoadPlugins(dir string) (map[string]jvspb.Validator, *multicloser.Closer, e
 
 		// Enable the plugin.
 		pluginClient := plugin.NewClient(&plugin.ClientConfig{
-			HandshakeConfig:  jvspb.Handshake,
+			HandshakeConfig: jvspb.Handshake,
+			// Plugins field need to be set otherwise exception will be thrown from
+			// https://github.com/hashicorp/go-plugin/blob/a88a423a8813d0b26c8e3219f71b0f30447b5d2e/client.go#L909
+			Plugins: map[string]plugin.Plugin{
+				// Due to [dispense name check], the name need to be consistent with the plugin name.
+				//
+				// [dispense name check]: https://github.com/hashicorp/go-plugin/blob/a88a423a8813d0b26c8e3219f71b0f30447b5d2e/grpc_client.go#L110
```

Comment:
Due to the code context, we can probably save this comment.

Reply:
Done and add all the details to the PR description instead